File: Assignments/Lab/Lab 01 - Oct 29 2020 Thursday/app/src/main/java/task/y34h1a/cse486/happybirthday/MainActivity.java
package task.y34h1a.cse486.happybirthday;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

import com.y34h1a.cse486.happybirthday.R;

public class MainActivity extends AppCompatActivity {
    public static final String TAG = "MainActivity";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.d(TAG, "OnCreate");
    }
}

File: Assignments/Lab/Lab 01 - Oct 29 2020 Thursday/app/src/main/java/task/y34h1a/cse486/happybirthday/MainActivity.kt
package task.y34h1a.cse486.happybirthday

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.y34h1a.cse486.happybirthday.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Log.d("MainActivity", "OnCreate");

    }
}